### DISCUSSION TITLE
Code Wars - Multiplication table for number

### DISCUSSION THREAD

**Discussion by ghost** [codewars/codewars.com#2437] [Q&A]
Why is my solution not being accepted? The output seems fine to me if not what am I doing wrong or what is it that I'm not understanding.

        int i;
	char *table[10];
	char str[10];
	char number[10];
	char answer[10];

	for(i = 0; i < 10; i++){
		table[i] = malloc(10 * sizeof(char));
	}
	
	for(i = 0; i < 10; i++){
		sprintf(str, "%d", i + 1);
		strcpy(table[i], str);
		strcat(table[i], " * ");
		sprintf(number, "%d", num);
		strcat(table[i], number);
		strcat(table[i], " = ");
		sprintf(answer, "%d", num * (i +1));
		strcat(table[i],answer);
	}

	for(i = 0; i < 10; i++){
		printf("%s\n", table[i]);
	}

**Comment by Blind4Basics** [comment]
Hello,

That's not the place for this. Please ask on the discourse page of that kata.
I don't practice this language, but from a quick glance, I guess you're printing the result to the console while CW's "way of solving" is instead to `return` the result.

Cheers
